.gitignore

```diff
@@ -130,3 +130,8 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+*.sqlite
+/.vs
+
+/reports/test-report.xml
+/package-lock.json
```

Comment:
please dont exclude this from commit. we expect npm install on the pipeline to follow the pinned package version from this file.